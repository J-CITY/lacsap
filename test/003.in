32.21e-10 3.1e+2


32


1222.121
