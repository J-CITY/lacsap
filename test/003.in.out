32.21e-10   0   0    literalFloat    25
   3.1e+2   0  10    literalFloat    25
       32   3   0      literalInt    26
 1222.121   6   0    literalFloat    25
